Sources/StreamChat/Database/DatabaseContainer.swift

```diff
@@ -153,6 +153,8 @@ class DatabaseContainer: NSPersistentContainer {
             }
         }
         
+        FetchCache.clear()
```

Comment:
Why is this needed? On init there should not be anything hanging on the cache. Is this maybe a fix for the tests?

Reply:
Yes, tests re-create the whole stack, starting from `ChatClient` but `FetchCache` was lingering around, having dirty cache.

This can also resolve a possible edge-case where 2 `ChatClient` instances are alive

Reply:
Interestingly enough, this opens the door for another question:

Should the DatabaseContainer hold a reference to the cache? It would make sense conceptually, as there should not be a shared cache across clients.

Reply:
That is something that might be out of the scope for this PR, as there is no easy way to do that now.
But, in any case, I would try not to have Test fixes in production code.